src/components/CommentList/__tests__/CommentList-test.js

```diff
@@ -37,4 +37,13 @@ describe('components/CommentList', () => {
         expect(wrapper).toMatchSnapshot();
         expect(wrapper.find('.ba-comment-list-item').length).toEqual(2);
     });
+
+    test('should scroll to the bottom on componentDidUpdate()', () => {
+        const wrapper = shallow(<CommentList comments={comments} onDelete={onDelete} />);
+        const instance = wrapper.instance();
+        instance.scrollToBottom = jest.fn();
+
+        instance.componentDidUpdate();
```

Comment:
My understanding is that this isn't an ideal way to test behavior tied to lifecycle methods. Doing something like `wrapper.setProps({ comments: someOtherComments })` would be closer to how the component actually behaves.